build-config/release/post-release-dependency-bump.sh

```diff
@@ -0,0 +1,39 @@
+# ---------------------------------------------------------------------------
```

Comment:
When are we running this? I mean which process calls this script?

Reply:
As of now, we decided to hold this. This was supposed to be called by the workflow itself